lib/glific/third_party/gcs/gcs.ex

```diff
@@ -177,4 +179,72 @@ defmodule Glific.GCS do
       )
     end
   end
+
+  @spec bucket_name(non_neg_integer()) :: String.t() | nil
+  defp bucket_name(org_id) do
+    case get_secrets(org_id) do
+      %{"bucket" => bucket} -> bucket
+      _ -> nil
+    end
+  end
+
+  @doc """
+  Enabling bucket logging for the specified organization
+  """
+  @spec enable_bucket_logs(non_neg_integer()) :: {:ok, String.t()} | {:error, String.t()}
+  def enable_bucket_logs(org_id) do
+    case bucket_name(org_id) do
+      nil ->
+        Logger.error("Bucket name not found for organization ID: #{org_id}")
+        {:error, "Bucket name not found"}
+
+      bucket_name ->
+        log_bucket = "#{bucket_name}-logs"
+        log_object_prefix = "log_object_prefix"
+
+        case update_bucket_logging(bucket_name, log_bucket, log_object_prefix) do
```

Comment:
just rename it to `do_enable_bucket_logs` right now it is confusing if u r enabling logs or updating bucket logging from name